pkgs/build-support/rust/hooks/cargo-setup-hook.sh

```diff
@@ -22,17 +22,17 @@ cargoSetupPostUnpackHook() {
         mkdir .cargo
     fi
 
-    config="$cargoDepsCopy/.cargo/config";
+    config="$cargoDepsCopy/.cargo/config.toml";
```

Comment:
Non‐blocking nit: If we’re changing this hook anyway we could get rid of this unnecessary `;`.

Reply:
can do

Reply:
updated/removed